src/Extractor/HttpClient.php

```diff
@@ -115,39 +139,34 @@ public function fetch($url, $skipTypeVerification = false, $httpHeader = [])
         $this->logger->log('debug', 'Trying using method "{method}" on url "{url}"', ['method' => $method, 'url' => $url]);
 
         try {
+            /** @var ResponseInterface $response */
             $response = $this->client->$method(
                 $url,
                 [
-                    'headers' => [
-                        'User-Agent' => $this->getUserAgent($url, $httpHeader),
-                        // add referer for picky sites
-                        'Referer' => $this->getReferer($url, $httpHeader),
-                    ],
-                    'timeout' => $this->config['timeout'],
```

Comment:
We can't configure timeout anymore?

Reply:
I think this should be defined somewhere in the README

Reply:
There is a paragraph `Timeout configuration` in the README